docs/sdks/client-sdks/javascript.md

```diff
@@ -70,16 +70,34 @@ If you are using the SDK for experiment assignments, make sure to pass in an ass
 
 ### Initialization options
 
-How the SDK fetches experiment configurations is configurable via additional optional initialization options:
-
-| Option | Description | Default |
-| ------ | ----------- | ------- | 
-| **`requestTimeoutMs`** (number) | Timeout in milliseconds for HTTPS requests for the experiment configurations. | `5000` |
-| **`numInitialRequestRetries`** (number) | Number of _additional_ times the initial configurations request will be attempted if it fails. This is the request typically synchronously waited (via `await`) for completion. A small wait will be done between requests. | `1` |
-| **`pollAfterSuccessfulInitialization`** (boolean) | Poll for new configurations (every 30 seconds) after successfully requesting the initial configurations. | `false` |
-| **`pollAfterFailedInitialization`** (boolean) | Poll for new configurations even if the initial configurations request failed. | `false` |
-| **`throwOnFailedInitialization`** (boolean) | Throw an error (reject the promise) if unable to fetch initial configurations during initialization. | `true` |
-| **`numPollRequestRetries`** (number) | If polling for updated configurations after initialization, the number of additional times a request will be attempted before giving up. Subsequent attempts are done using an exponential backoff. | `7` |
+How the SDK fetches, serves, and caches experiment configurations is configurable via additional optional initialization options:
+
+| Option                                            | Description                                                                                                                                                                                                                                                                                                                            | Default         |
```

Comment:
Because the table width changed, each line was "updated"--but for existing options this was just a chance to the internal spacing.